sources/academy/webscraping/scraping_basics_python/09_getting_links.md

```diff
@@ -199,8 +199,12 @@ def export_json(file, data):
     json.dump(data, file, default=serialize, indent=2)
 
 listing_url = "https://warehouse-theme-metal.myshopify.com/collections/sales"
-soup = download(listing_url)
-data = [parse_product(product) for product in soup.select(".product-item")]
+listing_soup = download(listing_url)
```

Comment:
I made this change because this way the code is better suited for the newly added lessons.